mavutil.py

```diff
@@ -264,7 +264,7 @@ def post_message(self, msg):
         if type == 'HEARTBEAT' and msg.get_srcComponent() != mavlink.MAV_COMP_ID_GIMBAL:
             self.target_system = msg.get_srcSystem()
             self.target_component = msg.get_srcComponent()
-            if float(mavlink.WIRE_PROTOCOL_VERSION) >= 1 and msg.type != mavlink.MAV_TYPE_GCS:
+            if float(mavlink.WIRE_PROTOCOL_VERSION) >= 1 and msg.type != mavlink.MAV_TYPE_GCS and msg.type != mavlink.MAV_TYPE_ONBOARD_CONTROLLER:
```

Comment:
if float(mavlink.WIRE_PROTOCOL_VERSION) >= 1 and msg.type not it (mavlink.MAV_TYPE_GCS, mavlink.MAV_TYPE_ONBOARD_CONTROLLER):